Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN mkdir -p ./src/bin && \
 COPY ./src ./src
 COPY ./components ./components
 
-RUN cargo build --no-default-features --release --features "jemalloc portable sse no-fail" 
+RUN make build
```

Comment:
seem we don't copy the Makefile to the docker before this command.

Have you run to verify that it can work?

Reply:
It builds a debug tikv, I think it's better to `make release`

Reply:
Friendly ping @xiamengyu

Reply:
@siddontang @overvenus Thank you for pointing out the problem, I just fixed it. Verified on my local MacOS. 
Please take another review.